core/templates/dev/head/filters.js

```diff
@@ -342,6 +342,26 @@ oppia.filter('summarizeNonnegativeNumber', [function() {
   };
 }]);
 
+// Note that this filter removes additional new lines or <p><br></p> tags 
```

Comment:
Looks like the linter failed, I'd guess it's due to the extra space at the end of this line.

You can check the linter locally by running `python scripts/pre_commit_linter.py`